spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -16,8 +16,19 @@
         }
       end
 
-      it 'returns the target' do
+      it 'returns the target and the target matches' do
         subject
+        target_matches = TargetMatchingService.new(Target.last).give_me_matches
+        target_matches = target_matches.map do |target|
```

Comment:
Can you try to avoid defininf this variable via a `.map` , and maybe just call `.to_json` to the previous line result?

Reply:
I tried that, but that result contains attributes like created date and last updated, that my response does not contain.